jsonschema/validators.py

```diff
@@ -312,6 +336,8 @@ def extend(validator, validators=(), version=None, type_checker=None):
     type_checker=_types.draft3_type_checker,
     version="draft3",
     id_of=lambda schema: schema.get(u"id", ""),
+    applicable_validators=_legacy_validators
```

Comment:
Let's call this `_legacy_validators.ignore_ref_siblings`.

Reply:
Done